ivy/functional/frontends/jax/numpy/statistical.py

```diff
@@ -65,6 +65,7 @@ def bincount(x, weights=None, minlength=0, *, length=None):
     return ret
 
 
+# reverse is not implemented here, whereas it is implemented in the actual framework
```

Comment:
I think we should be implementing it here too. Feel free to discuss your opinion here :)

Reply:
It is not implemented in the jax framework but in the tensorflow framework. So I think it should be implemented here. I did not implement the cumprod, function. I worked on the cummin function. I will remove this comment. Apologies for leaving this comment.

p.s The cummin function is implemented by jax and torch. it is not available in tensorflow and paddle. Hence i did not implement the exclusive argument